### ISSUE TITLE
Show how much each class contributes to total filesize

### ISSUE THREAD

**Issue by snakefoot** [icsharpcode/ILSpy#2517]
**Is your feature request related to a problem? Please describe.**
Trying to reduce the size and dependencies of an assembly. Would be great if ILSpy could show how much class / namespace contributed to the total filesize.

**Describe the solution you'd like**
Option to show source-code statistics in the class-tree. Ex. Line-Count + Binary Size

Ex:

- NameSpace (12 classes - 23456 lines - 789 bytes)
   - Classs (123 lines - 4 bytes)

**Comment by christophwille** [comment]
Use ilspycmd to decompile the assembly and let other tools create the statistic plus a nice html overview page (hint: LLMs)

**Comment by snakefoot** [comment]
Yes discovered that AOT-publish with this csproj-setting is useful for LLM to analyze:
```
<IlcGenerateDgmlFile>true</IlcGenerateDgmlFile>
```
Believe DGML-file is enabled by default with NET10 and newer.
